scripts/suscribeTransfer.js

```diff
@@ -0,0 +1,87 @@
+const WebSocket = require('ws')
```

Comment:
With this example snippet, we are able to subscribe to a ws endpoint and detect when new `tokenTransfers`, `tokenDeposits` and `tokenWithdrawns` occur.

cc: @ebma

Reply:
Nice, that's amazing 🎉